app/src/lib/editors/win32.ts

```diff
@@ -292,8 +316,12 @@ function isExpectedInstallation(
   switch (editor) {
     case ExternalEditor.Atom:
       return displayName === 'Atom' && publisher === 'GitHub Inc.'
+    case ExternalEditor.AtomBeta:
+      return displayName === 'Atom Beta' && publisher === 'GitHub Inc.'
+    case ExternalEditor.AtomNightly:
+      return displayName === 'Atom Nightly' && publisher === 'GitHub Inc.'  
     case ExternalEditor.VisualStudioCode:
-      return (
+      return ( 
```

Comment:
I think this bit of whitespace is causing the Prettier error - running `yarn lint:fix` over the branch and committing the changes should help get this passing CI again.

Reply:
Since I'm using Github Web I'll just remove the whitespace

Reply:
hmm, tried that but stil got issues

I don't have a command line editor so idk how I'd use Prettier @shiftkey

Reply:
also note that I haven't added everything required, just the framework

Reply:
so this patch is neither fully functional nor elegant

Reply:
@Rexogamer I pulled down the PR to see what's left and Prettier is suggesting these two changes:

<img width="946" src="https://user-images.githubusercontent.com/359239/61797432-c67e2e00-adfd-11e9-98cc-a20263514d3c.png">

Reply:
Okay, one mo

Reply:
@shiftkey any issues now should be functional ones

Reply:
yep, I looked at the error and I gather it's because I didn't finish it

Reply:
No problem, this area is what it's referring to:

https://github.com/desktop/desktop/blob/fcbb95d9df8741ab48f0dd06625c8204aa743e7c/app/src/lib/editors/win32.ts#L254-L260

Entries need to be added here for `AtomBeta` and `AtomNightly` (I think these both should behave the same).

Reply:
will I need to update the `atom.cmd` bit to `atom-nighty/atom-beta.cmd`?

Reply:
I don't believe they're renamed, but the quickest way to do this is to install these versions and check for yourself (which I'm doing at the moment). I'll report back in a bit.

Reply:
actualy I think I do; I checked Nightly and it is indeed `atom-nightly.cmd`